v1/plugin/session.go

```diff
@@ -0,0 +1,117 @@
+package plugin
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"time"
+
+	"net/http/pprof"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+var (
+	listenPort = "0"
+	LogLevel   = uint8(2)
+	pprofPort  = "0"
+)
+
+// Arguments passed to startup of Plugin
+type Arg struct {
+	// Plugin log level, see logrus.Loglevel
+	LogLevel uint8
+	// Ping timeout duration
+	PingTimeoutDuration time.Duration
+
+	// The listen port
+	ListenPort string
```

Comment:
ListenPort is used as a string on [line 122](https://github.com/intelsdi-x/snap-plugin-lib-go/pull/44/files#diff-01c2cbb747cc453d78a7b29d3e51c7d7R122).